### PR TITLE
Add new default "argon" skin

### PR DESCRIPTION
**Pull request by peppy**
- [x] Depends on #20354

Missing spinner, missing gradient on slider track (not even sure how to do this), missing border piece on slider reverse arrow. And probably a few other nuances. That said, I think it's ready to push this out and build on it as we go.

https://user-images.githubusercontent.com/191335/190975428-2bb854af-9151-48ca-b255-1dbcfa348956.mp4

Interested in feedback on the outer fill (currently only applied to hitcircles and not slider circles). Without it on circles, things feel too busy, but adding it to sliders as well feels a bit weird to my eye.

| Without | With |
|--|--|
| osu! 2022-09-19 at 07 13 07 | osu! 2022-09-19 at 07 15 58 |
| osu! 2022-09-19 at 07 12 40 | osu! 2022-09-19 at 07 15 46 |

Also to consider: should we force existing users across to argon if they are currently on triangles?

### CODE REVIEW

**Comment by mk56-spn** [comment]
Honestly it would be lovely to have a choice to not have slider gradients . It looks very clean right now. Iirc gradients are forced to avoid people making skins with no border and cheesing sliders by only going to slider ticks. But now that strict tracking is an option and anyone that really wants sliders to be followed to the letter has that it just feels not so necessary. I guess I just want to know if there's any chance we can get gradient-less sliders at some point

**Comment by peppy** [comment]
I have no idea where you are getting your rationale from, but it's not relevant to this thread, so please refrain from commenting here.

Customisation will be added to this new default skin in the future to allow fine-tuning it.

File: osu.Game/Skinning/SkinManager.cs [hunk 1]
```diff
@@ -49,24 +49,26 @@ public class SkinManager : ModelManager<SkinInfo>, ISkinSource, IStorageResource
...
-        };
+        public readonly Bindable<Live<SkinInfo>> CurrentSkinInfo = new Bindable<Live<SkinInfo>>(ArgonSkin.CreateInfo().ToLiveUnmanaged());
 
         private readonly SkinImporter skinImporter;
 
         private readonly IResourceStore<byte[]> userFiles;
 
         /// <summary>
-        /// The default skin.
+        /// The default "triangles" skin.
```
Comment (Feodor0090) [inline osu.Game/Skinning/SkinManager.cs:59]:
```suggestion
        /// The default "argon" skin.
```

Comment (peppy) [reply osu.Game/Skinning/SkinManager.cs]: I've just removed this as it's a private now.

**Comment by Feodor0090** [comment]
Choose classic skin. Close osu!. Open again, go to settings. Do not open open skin selection dropdown, use arrow-up key. I'm expecting triangles skin to be selected, but argon is selected instead. This happens only one time per run.

https://user-images.githubusercontent.com/53872073/191580173-04c27394-7ff8-4fa5-9e3f-cf5210d56911.mp4

**Comment by peppy** [comment]
@Feodor0090 please open a separate issue once this is merged. It's pretty minor and I don't foresee fixing it in this pull request (I've checked quickly and can't see why this would happen. may require a framework change to fix).

**Review by smoogipoo — APPROVED** [review]
As I said in PM, I'm not entirely sure about the playability of this skin:
- Judgements are too bright. Combined with the subpar cursor, streams ~~in kiai sections~~ are nigh unreadable (except in replays if you look at the cursor directly, which is not how people play).
- The white ring, on sliders specifically, makes the aim requirement seem more tight due to it being transparent.
- I'm not sure about accuracy w.r.t. the white ring, but my playstyle's not currently in a condition to evaluate this correctly.
- I feel like the game plays fine without the white ring in general, though with <100% dim there'd need to be something more under the hitobject - a shadow or something. Contrast isn't that big of a problem imo and could/should be handled in better ways (there's an issue/discussion for this somewhere). Removing it would resolve the issues brought up above (except judgements) and clear up the sizing;.
  - I also brought up that the ring could be redesigned and not part of the hit area - I have a feeling I've seen skins similar that use rings placed outside the hitcircle body, including things like cat ears on their circles.

That being said, I'm going to go with this for now and see what others (hi KoiFishu) think of this.

**Pull request merged**